src/test/ui/generic-associated-types/issue-93341.rs

```diff
@@ -1,5 +1,3 @@
-// check-pass
```

Comment:
This would represent a regression from stable to nightly. This code should be parsed without error.

Reply:
I fixed this already